examples/use-cases/SmartVF2SmartVF/README.md

```diff
@@ -0,0 +1,125 @@
+# Test Smart VF connection
+
+This example shows that NSC and NSE can work with each other over the SmartVF dual mode (kernel or dpdk) connection.
+
+## Requires
+
+Make sure that you have completed steps from [ovs](../../ovs) setup.
+
+## Run
+
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f ../namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Register namespace in `spire` server:
+```bash
+kubectl exec -n spire spire-server-0 -- \
+/opt/spire/bin/spire-server entry create \
+-spiffeID spiffe://example.org/ns/${NAMESPACE}/sa/default \
+-parentID spiffe://example.org/ns/spire/sa/spire-agent \
+-selector k8s:ns:${NAMESPACE} \
+-selector k8s:sa:default
+```
+
+Create customization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ${NAMESPACE}
+
+bases:
+- ../../../apps/nsc-kernel
+- ../../../apps/nse-kernel
+- ../../../apps/nsc-kernel-ponger
+
+
+patchesStrategicMerge:
+- patch-nsc.yaml
+- patch-nse.yaml
+EOF
+```
+
+Create NSC patch:
+```bash
+cat > patch-nsc.yaml <<EOF
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nsc-kernel
+spec:
+  template:
+    spec:
+      containers:
+        - name: nsc
+          env:
+            - name: NSM_NETWORK_SERVICES
+              value: kernel://icmp-responder/nsm-1?sriovToken=worker.domain/100G
```

Comment:
We have only `10G` NICs on our packet integration clusters, so this client will never start on them. Please use here `10G` instead of `100G`.

Reply:
@pperiyasamy  We should be able to CI the example provided... the packet.net env where we have HW to run such examples only has 10G nics... is it possible to get the example to conform to what we can actually test?  Feel free to talk in commentary for the example about how to alter it for MLX5 :)

Reply:
@edwarnicke currently the plan is to configure the MLX5 100G NIC as per this [instruction](https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/docs/ovs-offload.md#mellanox-sr-iov-configuation) and create `pci.config` file and pass it as `NSM_SRIOV_CONFIG_FILE` into ovs forwarder. I think once this configuration is automated  (like its done for legacy sriov VFs with sdk-sriov pcifunction) by enhancing sdk-sriov, then this can be easily integrated with packet.net, WDYT ?